Xwt.WPF/Xwt.WPFBackend/AlertDialogBackend.cs

```diff
@@ -57,6 +57,9 @@ public void Initialize (ApplicationContext actx)
 		public Command Run (WindowFrame transientFor, MessageDescription message)
 		{
 			this.icon = GetIcon (message.Icon);
+			if(string.IsNullOrEmpty(message.Title))
```

Comment:
One minor formatting change (space before parens) and one simplification of code.
```csharp
if (string.IsNullOrEmpty (message.Title))
    message.Title = transientFor?.Title ?? string.Empty;
```
```

Reply:
Also, please use `string` instead of `String` [details](https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html)